devmaster/modules/devshop/aegir_download/aegir_download.module

```diff
@@ -61,11 +61,10 @@ function hosting_task_download_form() {
  * Defines the list of tasks that appear under the gear icon.
  */
 function aegir_download_devshop_environment_menu($environment) {
-
-    if ($environment->site && $environment->site_status == HOSTING_SITE_ENABLED) {
-        $items[] = 'download';
-    }
-    return $items;
+  if ($environment->site && $environment->site_status == HOSTING_SITE_ENABLED) {
+    return ['download'];
+  }
+  return [];
```

Comment:
Is the return []  needed at all? I wonder if the module_invoke_all filters out empty values from hooks for us?